vhost_user_backend/src/lib.rs

```diff
@@ -462,7 +486,7 @@ struct VhostUserHandler<S: VhostUserBackend> {
     max_queue_size: usize,
     memory: Option<Memory>,
     vrings: Vec<Arc<RwLock<Vring>>>,
-    worker_thread: Option<thread::JoinHandle<VringWorkerResult<()>>>,
+    worker_thread: Option<Vec<thread::JoinHandle<VringWorkerResult<()>>>>,
```

Comment:
Don't need to use an `Option<Vec<T>>` as you can just use an empty vector for the equivalent of the None case.

Reply:
And it's plural now.